ui.js

```diff
@@ -157,3 +212,69 @@ document.addEventListener("DOMContentLoaded", function () {
   // modal functionality
   initializeModal();
 });
+
+
+let pakoLoaded = false;
+var pako = null;
+// Lazy load function that loads pako on the first call
+function loadPako() {
+  if (pako == null)
+    pako = BrowserCJS.require("https://cdnjs.cloudflare.com/ajax/libs/pako/2.0.4/pako.min.js");
+  return pako;
+}
+
+// Function to compress and decompress text, loading pako if necessary
+async function compressToBase64URL(text) {
+  const pako = loadPako(); // Ensure pako is loaded
+
+  // Compress the text
+  const compressed = pako.deflate(text, { to: 'string' });
+  const base64 = btoa(String.fromCharCode(...new Uint8Array(compressed)));
+
+  return base64.replace(/\+/g, '-').replace(/\//g, '_').replace(/=+$/, '');
+}
+
+async function decompressFromBase64URL(base64) {
+  const pako = loadPako(); // Ensure pako is loaded
+  // Decode the base64 URL
+  base64 = base64.replace(/-/g, '+').replace(/_/g, '/');
+  const compressed = Uint8Array.from(atob(base64), c => c.charCodeAt(0));
+  // Decompress the data
+  const decompressed = pako.inflate(compressed, { to: 'string' });
+
+  return decompressed;
+}
+
+function showTooltip(button, text) {
+  document.getElementById("tooltip").textContent = text;
+  // Position the tooltip near the button
+  const rect = button.getBoundingClientRect();
+  tooltip.style.top = `${rect.bottom + window.scrollY + 15}px`;
+  tooltip.style.left = `${rect.left + window.scrollX}px`;
+
+  // Show the tooltip
+  tooltip.classList.add('show');
+
+  // Hide the tooltip after 3 seconds
+  setTimeout(() => {
```

Comment:
why don't we just keep it there?

Reply:
hmm, never mind, I think this is the help window.

Reply:
because it will be annoying and you need to click something to make it disappear. This experience is aligned with many websites/applications that shows similar messages.